pkg/cli/image/manifest/manifest.go

```
@@ -118,6 +118,8 @@ func (o *SecurityOptions) Context() (*registryclient.Context, error) {
 }
 
 func (o *SecurityOptions) NewContext() (*registryclient.Context, error) {
+	os.Setenv("DISABLE_HTTP2", "1")
```

golangci-lint (2.12.2)

[error] Error return value of `os.Setenv` is not checked

(errcheck)